src/adapter/src/catalog.rs

```diff
@@ -1156,7 +1156,7 @@ impl Catalog {
         }
     }
 
-    #[tracing::instrument(name = "catalog::transact", level = "debug", skip_all)]
+    #[instrument(name = "catalog::transact", skip_all)]
```

Comment:
Intentional that this is debug now? Seems to be the same for many other cases.

Reply:
Isn't this info? debug used to be explicitly specified, but now no level is specified so it used the default of info, I think.